### PR TITLE
added outside click on currency toggle

### PR DESCRIPTION
**Pull request by EthanDann**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,38 +39,31 @@ const App = () => {
     localStorage.setItem(name, item);
   };
   const handleCurrency = (e) => {
```
Comment (EthanDann) [inline src/App.js:41]: This function isn't being called on click

File: src/App.js [hunk 2]
```diff
@@ -33,38 +39,31 @@ const App = () => {
...
         <BrowserRouter>
           <Navbar
             supportedCurrencies={supportedCurrencies}
             currencySymbol={currencySymbol}
-            isOpen={isOpen}
-            handleOpen={() => handleOpen()}
             activeCurrency={activeCurrency}
-            handleCurrency={() => handleCurrency()}
-            handleTextChange={() => handleTextChange()}
+            handleCurrency={handleCurrency}
```
Comment (EthanDann) [inline src/App.js:65]: Calling it right here

File: src/components/Navbar/CurrencyToggle/CurrencyToggle.js [hunk 3]
```diff
@@ -34,19 +49,19 @@ function CurrencyToggle(props) {
...
           </DownArrowContainer>
         </DropDownHeader>
-        {props.isOpen && (
+        {isOpen && (
           <DropDownListContainer>
             <DropDownList>
               {currencies.map((currency) => (
                 <ListItem
-                  onClick={props.handleCurrency}
+                  onClick={() => props.handleCurrency()}
```
Comment (EthanDann) [inline src/components/Navbar/CurrencyToggle/CurrencyToggle.js:64]: and right here

File: src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -23,11 +23,9 @@ function Navbar(props) {
         <CurrencyToggle
           supportedCurrencies={props.supportedCurrencies}
           currencySymbol={props.currencySymbol}
-          isOpen={props.isOpen}
-          handleOpen={props.handleOpen}
           activeCurrency={props.activeCurrency}
-          handleCurrency={props.handleCurrency}
-          handleTextChange={props.handleTextChange}
+          handleCurrency={() => props.handleCurrency()}
```
Comment (EthanDann) [inline src/components/Navbar/Navbar.js:27]: calling it here

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**